#!/usr/bin/python3
def raise_exception():
    i = "k" + 1
